### DISCUSSION TITLE
does this support buds 2 pro

### DISCUSSION THREAD

**Discussion by Aridea2021** [timschneeb/GalaxyBudsClient#358] [General]
I have ordered the new buds, but I do not know if this program can support the new buds 2 pro
so do you have any idea about that?

**Comment by timschneeb** [comment]
I have added untested support for them. However, I can't test it because I don't own the earbuds. 
It's possible that it won't work right out of the box, but you can try it out.

You can download a precompiled beta build on these pages:
* Windows: https://github.com/ThePBone/GalaxyBudsClient/actions/runs/2949009351
* Linux: https://github.com/ThePBone/GalaxyBudsClient/actions/runs/2949009328

**Reply by peleh** [reply]
Just tested on mine, it works on the Buds2 Pro. 

ANC on/off, ambient noise, gestures, tested those and the buds respond well to the configuration.

**Reply by PashaKagan** [reply]
> I have added untested support for them. However, I can't test it because I don't own the earbuds. It's possible that it won't work right out of the box, but you can try it out.
> 
> You can download a precompiled beta build on these pages:
> 
> * Windows: https://github.com/ThePBone/GalaxyBudsClient/actions/runs/2949009351
> * Linux: https://github.com/ThePBone/GalaxyBudsClient/actions/runs/2949009328

It says that the build files are expired...

**Reply by timschneeb** [reply]
Check the Actions tab for up-to-date builds. Here's the latest one for Windows: https://github.com/ThePBone/GalaxyBudsClient/actions/runs/3712399167

**Reply by ssp-iitr** [reply]
![Error_B2P](https://github.com/user-attachments/assets/0e5f5490-0181-4c10-ba72-88842bcd1c21)

Facing this error while Downgrading my Buds 2 Pro. Please help

**Comment by shubhamsri33** [comment]
how to downgrade galaxy buds 2 pro ?

**Comment by Aridea2021** [comment]
Autoreply：I have received your email and will reply as soon as possible.Best regards,[Dong]
